circus-train-avro/src/test/java/com/hotels/bdp/circustrain/avro/util/FileSystemPathResolverTest.java

```diff
@@ -0,0 +1,109 @@
+/**
+ * Copyright (C) 2016-2018 Expedia Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.bdp.circustrain.avro.util;
+
+import static org.junit.Assert.assertEquals;
+
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.Path;
+import org.apache.hadoop.hdfs.DFSConfigKeys;
+import org.junit.Test;
+
+public class FileSystemPathResolverTest {
+
+  private Configuration configuration = new Configuration();
+
+  @Test
+  public void resolveSchemeDoesntChangeSchemeForPathWithScheme() {
+    Path path = new Path("s3:/etl/test/avsc/schema.avsc");
+    assertEquals(path, new FileSystemPathResolver(configuration).resolveScheme(path));
```

Comment:
I think in general we prefer hamcrest: `assertThat(new FileSystemPathResolver(configuration).resolveScheme(path), is(path));` As you get nicer error reporting